### ISSUE TITLE
Slack message should include triggering user

### ISSUE THREAD

**Issue by paulbellamy** [weaveworks/flux#292]
And, maybe an optional comment message from them.

Need a way of determining which user, which probably means user-tokens.

wcloud fudged it a bit by defaulting to `$ whoami` with a `-u username` option for circle.

**Comment by paulbellamy** [comment]
Even better, on this would be determining the user who pushed the commits generating a new image, and through to the release, so you could do a slack message like:

```
flux:  @paul: Your changes to foo/bar, foo/baz, and foo/qox have been released to dev.
       When ready, click to [Release To Prod](https://cloud.weave.works/...)
```

**Comment by philwinder** [comment]
Similar to #501 and #274

**Comment by philwinder** [comment]
As a workaround, you could manually set the release user to be "@paul" so that when it's printed it will ping you.

But yes, unless you want to start recording usernames somewhere, this isn't going to be possible.

I'd recommend closing this and reopening under a larger umbrella issue regarding flux users.

**Comment by paulbellamy** [comment]
Or, could set up a slack alert on my shell username, that would work also and be easier.

I think this will be fixed by https://github.com/weaveworks/flux/pull/528

**Comment by philwinder** [comment]
Fixed in #528
